roles/run_hook/README.md

```diff
@@ -79,30 +89,23 @@ pre_infra_my_nice_hook:
     type: playbook
 ```
 
+### CR
 
-### CRD
+#### Single hook
 
-#### Single hook in its own parameter
-* `type`: (String) Type of the hook. In this case, set it to `crd`.
+* `type`: (String) Type of the hook. In this case, set it to `cr`.
 * `source`: (String) Source of the CRD. If it's a filename, the CRD is expected in `hooks/crds`. It can be an absolute path.
```

Comment:
Maybe we should put files in `hooks/crs` instead? For the sake of consistency.. ?

Reply:
Yes. Thank you. I should have checked the existing folder.
It should be fixed in the next commit.